marklogic-data-hub-central/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -46,14 +46,14 @@ const EntityTiles = (props) => {
           }
     }
 
-    const getMappingArtifactByMapName = async (entityTypeTitle,mapName) => {
+    const getMappingArtifactByMapName = async (entityTypeId,mapName) => {
```

Comment:
I'll try this out locally, but I was thinking that the UI needs to populate targetEntityType with entityTypeId instead of entityType too?

Reply:
I just tested this locally - I created a new mapping, and the targetEntityType was correctly set using the entityTypeId.